src/styles/blocks/day.scss

```diff
@@ -0,0 +1,25 @@
+.day {
+  display: flex;
```

Comment:
You do not have to add such styles to the items. It is enough for the container.

Reply:
it's necessary for centering of 
::before.
 display: flex;
  justify-content: center;
  align-items: center;

Reply:
you may centre content on elements with

line-height: $cell-size;
text-align: center;